part_01/PQHeap.java

```diff
@@ -19,7 +19,10 @@
     public PQHeap(int maxElms) {
         elements = new Element[maxElms - 1];
     }
-
+    /**
+     * This method extracts smallest Element from the array/heap and cut the size of the array/heap   
+     * @param return smallest Element after array has been minheapified  
+     */
     @Override
     public Element extractMin() {
         Element min = elements[0];
```

Comment:
Could use some comments explaining each individual line's function, so that's it's alomost too well-documented 🐤